Week7/MyFavoriteMovies/app/src/main/java/com/raywenderlich/myfavoritemovies/activities/MainActivity.kt

```diff
@@ -27,15 +28,40 @@ class MainActivity : AppCompatActivity() {
     private val movieRepository by lazy { MovieRepository() }
     private val userRepository by lazy { UserRepository() }
     private val movieAdapter by lazy { MovieAdapter(::movieItemClicked, ::movieItemLongClicked) }
+    private val remoteApi = App.remoteApi
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        getMovieFromApi("Star Wars: Episode I")
```

Comment:
Not sure why you only fetched movies by ID, but it would've been better if you used general API calls to get a list of movies e.g. by rating or top 20 movies or something similar. Also hardcoding movie names and all overall is bad, and I'm not sure why after all this there is a `fillMovieList()` call that supposedly does what I had described in the first point.